src/app/components/Skills/Wrapper.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import detailsIllustration from 'public/illustrations/details.svg';
+
+interface WrapperProps{
+    children: React.ReactNode;
+    as?:React.ElementType;
+    [key:string]:any;
+}
+
+const Wrapper: React.FC<WrapperProps> = ({children,as:AsComponent="div",...props})=> {
+    return(
+        <AsComponent
+            className = "bg-contain bg-left-top bg-no-repeat" {...props}
+            style={{backgroundImage: `url(${detailsIllustration.src})`}}
```

Comment:
This works, but one of the main things with Next.js is the Image Optimization, which we might as well use. They talk about using background images [here](https://nextjs.org/docs/app/building-your-application/optimizing/images#background-image). Let me know if you need some more help with this.